judo-ui-react/src/main/resources/actor/src/utilities/form-utils.ts.hbs

```diff
@@ -21,3 +21,23 @@ export const serviceDateToUiDate = (dateStr?: any) => {
   }
   return dateStr;
 };
+
+export const uiTimeToServiceTime = (time?: any | null): string | null => {
+    if(time === undefined || time === null) {
+        return null;
+    }
+
+    const resolved: Date = typeof time === 'string' ? new Date(time) : time;
```

Comment:
We need to ensure that we transform from and to UTC0

Reply:
[53e87a0](https://github.com/BlackBeltTechnology/judo-ui-react-template/pull/80/commits/53e87a03f9619102268770fb0a4fbae04f15c631)